docs/pages/docs/docs-theme/google-analytics.mdx

```diff
@@ -0,0 +1,65 @@
+import { Steps } from 'nextra/components'
```

Comment:
This doc related not only to docs theme, but to any theme

Reply:
So, would moving it under "Guide" be better?

Reply:
Under guide -> advanced